### PR TITLE
feat: add parameter state to repositories_client

### PR DESCRIPTION
**Pull request by btravouillon**
It is possible to ensure a repository does not exist with:

```
repositories:
    - { name: 'repo_to_delete', state: 'absent'}
```

See #455.

### CODE REVIEW

File: roles/core/repositories_client/readme.rst [hunk 1]
```diff
@@ -129,6 +130,7 @@ sources.list file.
 Changelog
 ^^^^^^^^^
 
+* 1.0.8: Add state parameter. Bruno Travouillon <devel@travouillon.fr>
```
Comment (oxedions) [inline roles/core/repositories_client/readme.rst:133]: Could you also increment variable repositories_client_role_version in vars ?

**Review by oxedions — CHANGES_REQUESTED** [review]
Dear @btravouillon 

Thanks a lot for this PR. 🙂 

Please see my single comment.

File: roles/core/repositories_client/readme.rst (hunk 1 above)
Comment (btravouillon) [reply roles/core/repositories_client/readme.rst:133]: 🤦‍♂️sure.

**Review by oxedions — APPROVED** [review]
Dear @btravouillon 

Thanks a lot for this PR 😊 

All is ok for me, merging.

**Pull request merged**